modules/components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -520,9 +520,9 @@ public JsonObject getPopularAssets(String type, String tenantId, int limit,
 		ResultSet resultSet;
 		JsonArray assets = new JsonArray();
 		JsonObject jsonObj = new JsonObject();
-		String tenantDomain = SocialUtil.getTenantDomain();
 
 		try {
+			String tenantDomain = SocialUtil.getTenantDomainFromTenantId(Integer.parseInt(tenantId));
```

Comment:
Why you are not using the standard way of getting the tenant domain. There is a MultitenantUtil class AFAIK and it should be used whenever possible.